packages/vue/src/components/organisms/SfCollectedProduct/SfCollectedProduct.stories.js

```diff
@@ -24,23 +24,28 @@ storiesOf("Organisms|CollectedProduct", module)
         default: text("title", "Product name", "Props")
       },
       regularPrice: {
-        default: text("regularPrice", "$10,99", "Props")
+        default: text("regular-price", "$10,99", "Props")
```

Comment:
For the docs-auto-generation, the knobs name needs to be the same as the prop name. Otherwise it will show wrong prop names in the "common usage" section of the components docs.

Reply:
ok, I miss this thing, sorry for that :)

Reply:
no prop! I mean: No Prob 😁

Reply:
fixed: https://github.com/DivanteLtd/storefront-ui/pull/485/commits/ae5c3119beb2cddf8b0863bf38b7bec352581050